pubsub/azure/servicebus/topics/servicebus.go

```diff
@@ -76,7 +76,7 @@ func (a *azureServiceBus) Publish(req *pubsub.PublishRequest) error {
 	ebo := backoff.NewExponentialBackOff()
 	ebo.InitialInterval = time.Duration(a.metadata.PublishInitialRetryIntervalInMs) * time.Millisecond
 	bo := backoff.WithMaxRetries(ebo, uint64(a.metadata.PublishMaxRetries))
-	bo = backoff.WithContext(bo, a.publishCtx)
+	bo = backoff.WithContext(bo, ctx)
```

Comment:
Same comments as in the previous file:

1. Please remove the now-unused `a.publishCtx` property from the struct
2. Please use name `publishCtx, publishCancel` rather than `newCtx`

Reply:
Done.